src/main/java/tool/arenasizer/CapacityPanel.java

```diff
@@ -0,0 +1,185 @@
+
+        // VIP
+        labelVipBoxes = new JLabel();
+        loadImageIcon("gui/bilder/arena/vipboxes.png").ifPresent(labelVipBoxes::setIcon);
+        labelVipBoxes.setHorizontalTextPosition(SwingConstants.LEADING);
+        labelVipBoxes.setHorizontalAlignment(HORIZONTAL_ALIGNMENT_PLACE_TYPE);
+        add(labelVipBoxes);
+        labelCountVipBoxes = new LabelWithSignedNumber(EMPTY, HORIZONTAL_ALIGNMENT_PLACES);
+        add(labelCountVipBoxes);
+        labelPercentageVipBoxes = new LabelWithSignedNumber(EMPTY, HORIZONTAL_ALIGNMENT_PERCENTAGE);
+        add(labelPercentageVipBoxes);
+
+        // Total
+        labelTotal = new JLabel(EMPTY, HORIZONTAL_ALIGNMENT_PLACE_TYPE);
+        setFontToBold(labelTotal);
+        add(labelTotal);
+        labelCountTotal = new LabelWithSignedNumber(EMPTY, HORIZONTAL_ALIGNMENT_PLACES);
+        setFontToBold(labelCountTotal);
+        add(labelCountTotal);
+        labelPercentageTotal = new LabelWithSignedNumber(EMPTY, HORIZONTAL_ALIGNMENT_PERCENTAGE);
+        setFontToBold(labelPercentageTotal);
+        add(labelPercentageTotal);
+
+        // Misc
+        label1 = new JLabel(EMPTY, HORIZONTAL_ALIGNMENT_PLACE_TYPE);
+        setFontToBold(label1);
+        add(label1);
+        label2 = new JLabel(EMPTY, HORIZONTAL_ALIGNMENT_PLACES);
+        setFontToBold(label2);
+        add(label2);
+        label3 = new JLabel(EMPTY, HORIZONTAL_ALIGNMENT_PERCENTAGE);
+        setFontToBold(label3);
+        add(label3);
+
+        setTranslation();
+    }
+
+    public void setTitle(String titel) {
+        setBorder(BorderFactory.createTitledBorder(titel));
+    }
+
+    public void setTranslation() {
+        labelPlaceType.setText(getLanguageString("ArenaInfoPanel.place_type"));
+        labelPlaces.setText(getLanguageString("ArenaInfoPanel.places"));
+        labelPercentage.setText(getLanguageString("ArenaInfoPanel.rate_in_percent"));
+
+        labelTerraces.setToolTipText(getLanguageString("ls.club.arena.terraces"));
+        if (LONG_MODE) {
+            labelTerraces.setText(labelTerraces.getToolTipText());
+        }
+
+        labelBasicSeating.setToolTipText(getLanguageString("ls.club.arena.basicseating"));
+        if (LONG_MODE) {
+            labelBasicSeating.setText(labelBasicSeating.getToolTipText());
+        }
+
+        labelSeatsUnderRoof.setToolTipText(getLanguageString("ls.club.arena.seatsunderroof"));
+        if (LONG_MODE) {
+            labelSeatsUnderRoof.setText(labelSeatsUnderRoof.getToolTipText());
+        }
+
+        labelVipBoxes.setToolTipText(getLanguageString("ls.club.arena.seatsinvipboxes"));
+        if (LONG_MODE) {
+            labelVipBoxes.setText(labelVipBoxes.getToolTipText());
+        }
+
+        labelTotal.setToolTipText(getLanguageString("Gesamtgroesse"));
+        if (LONG_MODE) {
+            labelTotal.setText(labelTotal.getToolTipText() + " " + SUM_SIGN);
+        } else {
+            labelTotal.setText(SUM_SIGN);
```

Comment:
I wonder if the sum sign should be part of the localised/translation strings: I am thinking in particular of languages that are RTL rather LTR, this may cause some strange things?

Reply:
Review Finding 3: Add sum sign to translated texts